examples/with-vue-nuxt/pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
downgrade?

Reply:
I, uh, don't have an explanation there. I'm going to let pnpm lead the way.